h2o-py/h2o/grid/__init__.py

```diff
@@ -5,7 +5,8 @@
 from .grid_search import H2OMultinomialGridSearch
 from .grid_search import H2ODimReductionGridSearch
 from .grid_search import H2ORegressionGridSearch
+from .infogram_grid import infogram_grid
 
 __all__ = ['H2OGridSearch', 'H2OBinomialGridSearch', 'H2OClusteringGridSearch',
            'H2OAutoEncoderGridSearch', 'H2OMultinomialGridSearch',
-           'H2ODimReductionGridSearch', 'H2ORegressionGridSearch']
+           'H2ODimReductionGridSearch', 'H2ORegressionGridSearch', "infogram_grid"]
```

Comment:
infogram_grid is different than names we have been using.

Reply:
Yes, thank you. I am considering using object instead of a function but since the logic is very simple I don't really want to complicate it.